blueprint/main.tf

```diff
@@ -111,3 +111,66 @@ resource "kubernetes_namespace" "test" {
   }
   provider = kubernetes.test
 }
+
+resource "kubernetes_deployment" "team-1-app" {
+  provider = kubernetes.dev
+
+  metadata {
+    name = "group16-team1-test-app"
+    labels = {
+      test = "MyExampleApp"
+    }
+    namespace = "namespace-${var.team_id}"
+  }
+
+  spec {
+    replicas = 2
+
+    selector {
+      match_labels = {
+        test = "MyExampleApp"
+      }
+    }
+
+    template {
+      metadata {
+        labels = {
+          test = "MyExampleApp"
+        }
+      }
+
+      spec {
```

Comment:
Do you think there is a way to import this from a YAML file?  Most people will probably prefer to do their deployments in regular k8s yaml, I bet.

Reply:
It seems we can't do it using the basic kubernetes provider: https://stackoverflow.com/questions/49511156/terraform-kubernetes-provider-spec-from-file

Reply:
I'd advise https://github.com/ericchiang/terraform-provider-k8s, I think it's probably worth using a non-basic provider for this, as long as it's in the terraform registry.

Reply:
Hm, I guess it's not... Hm!